packages/project-access/src/project/specification.ts

```diff
@@ -187,15 +186,15 @@ async function getSpecificationVersion(root: string, options?: { logger?: Logger
  */
 export async function getSpecificationPath(root: string, options?: { logger?: Logger }): Promise<string> {
     const logger = options?.logger;
-    const moduleName = '@sap/ux-specification';
+    const moduleName = join('@sap', 'ux-specification');
```

Comment:
Not sure about this, the module name is `@sap/ux-specification` no matter which operating system. The fact we use this to store it under path `join(scope, name)` is purely internal. The function `getModulePath()` expects the name.
To avoid confusion I'd use `join()` later in this function when we do `slice`

Reply:
@Klaus-Keller done with https://github.com/SAP/open-ux-tools/pull/2149/commits/0ce1cf3a0a54e478d3625d9d852889d082ef68fe, would that work?